### PR TITLE
Add docs for OpenStack

### PR DESCRIPTION
**Pull request by akhil-rane**
Jira: https://issues.redhat.com/browse/CO-1090

### CODE REVIEW

**Comment by akhil-rane** [comment]
/hold
This is yet to be tested. Waiting for the openstack environment to get ready.

**Comment by akhil-rane** [comment]
This is yet to be tested. Waiting for the openstack environment to get ready.

File: README.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ The Hive service can be used to provision and perform initial configuration of O
 * AWS
 * Azure
 * Google Cloud Platform
+* oVirt
+* vSphere
+* OpenStack
```
Comment (joelddiaz) [inline README.md:17]: put them in alphabetical order

File: docs/using-hive.md [hunk 2]
```diff
@@ -311,6 +325,27 @@ platform:
     vCenter: vcenter.example.com
 ```
 
+For Openstack, replace the contents of `compute.platform` with:
```
[the remark this answers is not in the document]
Comment (akhil-rane) [reply docs/using-hive.md:328]: just added a note below to use correct instance type

**Comment by akhil-rane** [comment]
/cancel hold
Tested with hive on openstack. Looks good.

**Comment by akhil-rane** [comment]
/hold cancel
Tested with hive on openstack. Looks good.

File: docs/using-hive.md [hunk 3]
```diff
@@ -311,6 +325,29 @@ platform:
...
+    type: ci.m4.xlarge
+```
+Note: Use an instance type that meets the minimum requirement for the version of OpenShift being installed.
+
+and replace the contents of `platform` with:
+```yaml
+  openstack:
+    cloud: mycloud
+    computeFlavor: m1.large
+    externalNetwork: provider_net_shared_3
```
Comment (joelddiaz) [inline docs/using-hive.md:347]: can we turn this into a more generic-looking value? something like "openstack_network_name"

**Review by joelddiaz — COMMENTED** [review]
one small nit. otherwise looks good

File: docs/using-hive.md (hunk 3 above)
Comment (akhil-rane) [reply docs/using-hive.md]: done

**Review by joelddiaz — COMMENTED** [review]
/lgtm

**Pull request merged**